src/test/java/com/google/javascript/gents/multiTests/provide_imports/default_exports2_keep.js

```diff
@@ -0,0 +1,2 @@
+goog.provide('d.export2');
```

Comment:
I think you wanted this to be 'goog.module'. I am even surprised this works?!? It is not valid closure, AFAIK.

Reply:
Done.

It's 6 test cases now:
{goog.module, goog.provide} × {no default exports, only default exports, both default and non-default exports}